src/state_machine/hashmap.rs

```diff
@@ -0,0 +1,62 @@
+#![cfg(feature="hashmap")]
```

Comment:
It may be... Without it though how would the reader know this was only loaded on the feature?